### PR TITLE
Presentation revealjs update

### PR DESCRIPTION
**Pull request by hroyrh**
Added steps to statically test presentation using the revealjs framework and removed a few unclear steps.

### CODE REVIEW

**Comment by kashifest** [comment]
/hold
Until default branch of the repo is main

**Review by lentzi90 — COMMENTED** [review]
Nice! I wasn't even aware of this presentation template!

I would suggest adding a `.gitignore` in the presentations folder for avoiding issues with the `dist` and `plugin` folders.

One nit below, other than that this looks good to me! :slightly_smiling_face:

**Comment by hroyrh** [comment]
Thanks for the review @lentzi90 , I have applied the suggestions. Please take a look and see if everything looks okay.

**Review by lentzi90 — APPROVED** [review]
Very nice!

**Comment by hroyrh** [comment]
@fmuyassarov can you please take a look and merge if everything looks okay.

**Comment by lentzi90** [comment]
I know @fmuyassarov will be away a few weeks, so will to try to assign to someone else from the owners list.
/assign @hardys

File: docs/presentations/README.md [hunk 1]
```diff
@@ -20,20 +20,32 @@ ls metal3-docs/docs/presentations/test-presentation
 test-image1.png  test-image2-capi.png test-presentation.html
 ```
 
-To use this with revealjs, follow these steps (for more details refer
-  [here](https://revealjs.com/installation/#full-setup)) :
+To test your presentation with the revealjs framework, there are two simple
+options :
```
Comment (furkatgofurov7) [inline docs/presentations/README.md:24]:
```suggestion
options:
```

File: docs/presentations/README.md [hunk 2]
```diff
@@ -20,20 +20,32 @@ ls metal3-docs/docs/presentations/test-presentation
...
+
+1. Copy the `dist` and `plugin` directories from revealjs respository to the presentations directory.
```
Comment (furkatgofurov7) [inline docs/presentations/README.md:26]:
```suggestion
1. Copy the `dist` and `plugin` directories from revealjs repository to the presentations directory.
```

File: .gitignore [hunk 3]
```diff
@@ -6,4 +6,10 @@ design/*.toc.*
 hack/mdbook
 # Ignore mdbook boilerplate
 # from being committed.
-docs/user-guide/book
\ No newline at end of file
+docs/user-guide/book
+
+# Ignore reveals plugin
```
Comment (furkatgofurov7) [inline .gitignore:11]:
```suggestion
# Ignore revealjs plugin
```

File: docs/presentations/README.md [hunk 4]
```diff
@@ -20,20 +20,32 @@ ls metal3-docs/docs/presentations/test-presentation
...
+Here is an example :
 
 ```bash
+
 ## Clone revealjs repository
+git clone https://github.com/hakimel/reveal.js.git
 
-cd ${to_your_presentation_directory} && git clone https://github.com/hakimel/reveal.js.git
+## Option 1
+cd ${to_your_presentation_directory}
```
Comment (furkatgofurov7) [inline docs/presentations/README.md:37]: For consistency
```suggestion
cd ${your_presentation_directory}
```

**Review by furkatgofurov7 — COMMENTED** [review]
Thanks for the PR, please find few suggestions below:

**Comment by hroyrh** [comment]
@furkatgofurov7 thanks for the review. I've submitted the suggested changes, please take a look and see if everything looks good.

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Comment by dtantsur** [comment]
/approve

I don't really like copying files around, but works for now.

**Pull request merged**